### ISSUE TITLE
Manually fix buggy invoice

### ISSUE THREAD

**Issue by nickoferrall** [ParabolInc/parabol#6826]
Stripe only allows 250 invoice items per invoice and each seat change count for 2. So, when we recently tried to remove 278 org users, the invoice items got stuck around 125 add/removes.

Next steps:

- short term: manually process those invoice items
- medium term: filter out invoice items that don't match the invoice's date range
- long term: special case handling when there are >= 150 invoice items (created a "other users" bucket)

See [this Slack thread](https://parabol.slack.com/archives/C836NA350/p1655858819881809) for more info.

### AC

- [ ] Manually fix Parabol's invoice so that the invoice line items reflect the org users

**Comment by jordanh** [comment]
About many customers are affected? Would the revenue implication be for a single month, or if we do not fix this will their subscription totals be incorrect in perpetuity?

**Comment by nickoferrall** [comment]
> About many customers are affected? 

Only Parabol 

> Would the revenue implication be for a single month, or if we do not fix this will their subscription totals be incorrect in perpetuity?

The way we think it works is Stripe creates an invoice with a max total of 250 items. If there are more than 250 items, the remainder are added to the upcoming invoice. The following month's invoice will include these upcoming items, which means the invoice for July, for example, can include seat changes from May.


This was fixed by speaking with Stripe and getting the invoice item limit increased from 250 to 2,000. I had closed issue https://github.com/ParabolInc/parabol/issues/6810, but will also close this one now.
